articles/install-guide/update.md

```diff
@@ -100,13 +100,13 @@ Select your development environment below.
 
 ### Update IQ# for Python
 
-1. Update the `iqsharp` kernel
+1. Update the `iqsharp` kernel 
 
     ```bash
     dotnet tool update -g Microsoft.Quantum.IQSharp
     dotnet iqsharp install
     ```
-	If after the second command you see an error message `Could not install iqsharp into your Jupyter configuration...`, you can keep following the steps below and enter the command again after step 3. 
+	If after the second command you see an error message `Could not install iqsharp into your Jupyter configuration...`, you can keep following the steps below and enter the command again after step 3.
```

Comment:
I encountered the error my first time updating. Found this was a workaround, so I threw the line in just in case it happens to others. Unsure why the error happened, but with the next update it didn't---can't tell if it's fixed or just had to do with that particular update. 

Any insight?

Reply:
I'm not sure I could diagnose from that alone, sorry. Was there anything else printed in the log?

Reply:
It was only about two lines in whole, but I forgot to make a full note of it---my apologies there. I just repeated the process a few times in different virtual envs, but couldn't get the error to happen again. Of course, that's actually a good thing. 
I'm still unsure if it was merely a one-off on my part, though, so I'll leave it up to you if we want to keep this line in or not. Just let me know.

Reply:
No worries! I'd suggest dropping this bit for now, in any case; it's easy enough to add back in once we can better diagnose the error. Thanks!